tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/klass/ServiceClassTest.java

```diff
@@ -43,8 +43,12 @@ public void testBasicStructAsObject() {
 
     @Test
     public void testResourceMethodsDoesNotAffectAssignability() {
-        CompileResult compileResult = BCompileUtil.compile("test-src/klass/resource-method-assignability-test.bal");
-        BRunUtil.invoke(compileResult, "testServiceObjectValue");
+        CompileResult result = BCompileUtil.compile("test-src/klass/resource-method-assignability-negative.bal");
```

Comment:
We usually include `test` in the file.
```suggestion
        CompileResult result = BCompileUtil.compile("test-src/klass/resource-method-assignability-negative-test.bal");
```

Not introduced by this PR, but we use underscores in bal file names.

Reply:
Yes, will update